internal/rbd/rbd_attach.go

```diff
@@ -193,27 +194,30 @@ func waitForPath(ctx context.Context, pool, namespace, image string, maxRetries
 	return "", false
 }
 
-// Check if rbd-nbd tools are installed.
-func checkRbdNbdTools() bool {
+// setRbdNbdToolFeatures set features available with rbd-nbd.
+func setRbdNbdToolFeatures() {
 	// check if the module is loaded or compiled in
+	hasNBD = true
```

Comment:
Default is `false`, this temporary changes it to `true`, after which it may become `false` again. That isn't very clean. Better only set it once to `true` after successful detection.

Reply:
sure